### DISCUSSION TITLE
file not found error

### DISCUSSION THREAD

**Discussion by Raysyu** [meteostat/meteostat-python#96] [Q&A]
Dear meteostat,

I'm using your API to reteive weather data and this happened.
![image](https://user-images.githubusercontent.com/52298697/169832994-cdc7e1d8-5f98-40f5-a455-ac364bb66047.png)
I was running this locally with a small portion of the original data about 10000/17500000 rows and it's working and as I ran the original file in a server this happened. 

I would assume the locaiton of 'cache' can't be foudn when executing `self.clear_cache`
not sure what to do but any help would be great!

**Comment by clampr** [comment]
Hi, I guess your assumption is correct. You can try to specify a different `cache_dir` manually (https://dev.meteostat.net/python/api/hourly/#attributes). However, we could probably wrap this in a `try ... except` and show a warning instead.

**Reply by Raysyu** [reply]
Hi, thanks for replying, I guess my case happened quite rare it was solved after rebooting the device… quite silly of me but yeah, a bit weird. Also I guess I was running two programs simultaneously to retrieve maybe one of the program cleared the cache then the other one left with nothing so the error occurred.
